test/chpldoc/errors/saveSphinxInDocs.doc.compopts

```diff
@@ -0,0 +1 @@
+--save-sphinx docs
```

Comment:
I believe this should be a .doc.chpldocopts file

Reply:
I wondered about that, but it seemed to work as-is:
```bash
[Executing compiler /Users/bradc/chapel/bin/darwin-x86_64/chpldoc --save-sphinx docs saveSphinxInDocs.doc.chpl < /dev/null]
```

Reply:
They may be functionally the same, but I think it helps to follow the trend of other chpldoc tests.  "Compilation options" isn't really meaningful for chpldoc, you're not really "compiling" though it is using some initial passes from the Chapel compler, there's no corresponding "execution options", and a .execopts file would never be run even if start_test let you write one without complaining about it.  Whereas .chpldocopts is specifically "chpldoc options".

Reply:
I'll change it, but I think if you're going to insist on that approach, `sub_test` really shouldn't be using the .compopts file (and ideally should warn the user that it's ignoring it and that they should use a chpldocopts file instead) rather than using it and then relying on developers and the review process to enforce that policy.

Reply:
I think that's fair, and honestly was surprised it was allowed myself

Reply:
I conformed and filed https://github.com/chapel-lang/chapel/issues/16910